### PR TITLE
#16 improve the save task button

### LINKED ISSUE #16: improve the save task button
- increase save text and the right place to be able to click easily, use  default  text sizes

- once pressing enter on the task description  it should go to save button automatically add next  focus there

- write  it in as much less code as possible

### PR DESCRIPTION
**Pull request by meetkparmar**
Fixed #16.

### CODE REVIEW

**Comment by meetkparmar** [comment]
Could you please tell me how to fix this? It seems to issue of google-services.json set in CI. @prudhvir3ddy

File: app/src/main/java/com/prudhvir3ddy/todo_app_gettingthingsdone/view/main/BottomSheetDialog.kt [hunk 1]
```diff
@@ -22,6 +26,15 @@ class BottomSheetDialog : BottomSheetDialogFragment() {
   ): View? {
 
     val v = inflater.inflate(layout.dialog, container, false)
+
+    v.task_desc_et.setOnEditorActionListener { v, actionId, event ->
```
Comment (prudhvir3ddy) [inline app/src/main/java/com/prudhvir3ddy/todo_app_gettingthingsdone/view/main/BottomSheetDialog.kt:30]: we can shift this to a new method ?

File: app/src/main/res/layout/dialog.xml [hunk 2]
```diff
@@ -44,7 +45,7 @@
       />
   <TextView
       android:id="@+id/save_tv"
-      style="@style/TextAppearance.AppCompat.Small"
+      style="@style/TextAppearance.AppCompat.Medium"
```
Comment (prudhvir3ddy) [inline app/src/main/res/layout/dialog.xml:48]: what do you think about this ?
looks good, matches our functionality

https://material.io/develop/android/components/buttons/#text-button

**Comment by prudhvir3ddy** [comment]
@meetkparmar  can you commit a dummy google-services.json file ?

just leave all details to empty and let the package-name be same.

we can take this to some other MR, i can merge this MR without ci passing

File: app/src/main/java/com/prudhvir3ddy/todo_app_gettingthingsdone/view/main/BottomSheetDialog.kt (hunk 1 above)
Comment (meetkparmar) [reply app/src/main/java/com/prudhvir3ddy/todo_app_gettingthingsdone/view/main/BottomSheetDialog.kt:30]: Ok

File: app/src/main/res/layout/dialog.xml (hunk 2 above)
Comment (meetkparmar) [reply app/src/main/res/layout/dialog.xml:48]: Looks good!

**Comment by prudhvir3ddy** [comment]
@meetkparmar if you are working on this then rebasing will fix the ci

**Comment by meetkparmar** [comment]
I have created a new PR #28. So, closing this one. @prudhvir3ddy

**Pull request closed without merging**